app/models/opensearch.rb

```diff
@@ -17,10 +17,10 @@ def default_index
   # Construct the json query to send to elasticsearch
   def build_query(from)
     query_hash = {
-      from: from,
+      from:,
       size: SIZE,
-      query: query,
-      aggregations: aggregations
+      query:,
+      aggregations:
```

Comment:
I suspect that this is a syntactic sugar in Rails that I've forgotten - but the change in this hash is because the `query` and `aggregations` methods are called implicitly, and `from` is populated auto-magically?

Reply:
Correct. When I changed the target ruby version in rubocop, this changed automatically when I save the file and seems to be the new preferred way of handling this. If we find we don't like it, we can decide to ignore the new rule. Other than being surprised it now works like this, I think I can learn to appreciate it.